src/css/support.js

```diff
@@ -46,61 +46,57 @@ define([
 		documentElement.removeChild( container );
 	}
 
-	// Support: node.js jsdom
-	// Don't assume that getComputedStyle is a property of the global object
```

Comment:
This should be safe, see 3722aef8be5d31a7dbf3de6998cdf249081f028c , dc3f7d3868e00fb318069b6f87dfc1fc4d10e56f and http://bugs.jquery.com/ticket/12235 for background.